docs/index.md

```diff
@@ -78,7 +78,7 @@ Add-Content -Path $PROFILE -Value '(& pixi completion --shell powershell) | Out-
 ### Fish
 
 ```fish
-echo 'pixi completion --shell fish | source' >> ~/.config/fish/config.fish
+pixi completion --shell > ~/.config/fish/completions/pixi.fish
```

Comment:
just one thing to note - this changes the completions to be "non-dynamic", right?
ie. someoen might have to update their completions file after a pixi release.

Reply:
Hmm you are right. Let me test something.